paddle/phi/api/yaml/op_compat.yaml

```
@@ -367,6 +367,13 @@
   extra :
     attrs : [bool use_mkldnn = false]
 
+- op : bilateral_slice
+  backward : bilateral_slice_grad
+  inputs :
+    {x:X, grid:Grid, guide:Guide}
```

```suggestion
    {x: X, grid: Grid, guide: Guide}
```
问题应该是因为这个：https://stackoverflow.com/questions/42124227/why-does-the-yaml-spec-mandate-a-space-after-the-colon